README.md

```diff
@@ -94,114 +115,122 @@ fi
 
 Default:
 
-```
+```bash
 startGroup "Print opam config"
   opam config list; opam repo list; opam list
 endGroup
 ```
 
-See [`custom_script`](#custom_script) for more details.
+See [`custom_script`](#custom_script) and [startGroup/endGroup](#startGroupendGroup) for more details.
 
-#### `install`
+#### `install` *Optional* 
 
-*Optional* The bash snippet to install the `opam` `PACKAGE` dependencies.
+The bash snippet to install the `opam` `PACKAGE` dependencies.
 
 Default:
 
-```
+```bash
 startGroup "Install dependencies"
   opam pin add -n -y -k path $PACKAGE $WORKDIR
   opam update -y
   opam install -y -j 2 $PACKAGE --deps-only
 endGroup
 ```
 
-See [`custom_script`](#custom_script) for more details.
+See [`custom_script`](#custom_script) and [startGroup/endGroup](#startGroupendGroup) for more details.
 
-#### `after_install`
+#### `after_install` *Optional* 
 
-*Optional* The bash snippet to run after `install` (if successful).
+The bash snippet to run after `install` (if successful).
 
 Default:
 
-```
+```bash
 startGroup "List installed packages"
   opam list
 endGroup
 ```
 
-See [`custom_script`](#custom_script) for more details.
+See [`custom_script`](#custom_script) and [startGroup/endGroup](#startGroupendGroup) for more details.
 
-#### `before_script`
+#### `before_script` *Optional* 
 
-*Optional* The bash snippet to run before `script`. Default `""` (empty string).
+The bash snippet to run before `script`.
 
-See [`custom_script`](#custom_script) for more details.
+Default: `""` (empty string).
 
-#### `script`
+See [`custom_script`](#custom_script) and [startGroup/endGroup](#startGroupendGroup) for more details.
 
-*Optional* The bash snippet to install the `opam` `PACKAGE`.
+#### `script` *Optional* 
+
+The bash snippet to install the `opam` `PACKAGE`.
 
 Default:
 
-```
+```bash
 startGroup "Build"
   opam install -y -v -j 2 $PACKAGE
   opam list
 endGroup
 ```
 
-See [`custom_script`](#custom_script) for more details.
+See [`custom_script`](#custom_script) and [startGroup/endGroup](#startGroupendGroup) for more details.
+
+#### `after_script` *Optional* 
 
-#### `after_script`
+The bash snippet to run after `script` (if successful).
 
-*Optional* The bash snippet to run after `script` (if successful). Default `""` (empty string).
+Default: `""` (empty string).
 
-See [`custom_script`](#custom_script) for more details.
+See [`custom_script`](#custom_script) and [startGroup/endGroup](#startGroupendGroup) for more details.
 
-#### `uninstall`
+#### `uninstall` *Optional* 
 
-*Optional* The bash snippet to uninstall the `opam` `PACKAGE`.
+The bash snippet to uninstall the `opam` `PACKAGE`.
 
 Default:
 
-```
+```bash
 startGroup "Uninstallation test"
   opam remove $PACKAGE
 endGroup
 ```
 
-See [`custom_script`](#custom_script) for more details.
+See [`custom_script`](#custom_script) and [startGroup/endGroup](#startGroupendGroup) for more details.
 
-#### `custom_script`
+#### `custom_script` *Optional* 
```

Comment:
Same remarks as [above](https://github.com/coq-community/docker-coq-action/pull/56/files#r627811092) for other headings.

Reply:
It's kind of ugly, another option is that the first line under each heading would have something like the following format:

```
[optional|required] default: 
```

Reply:
Hi @Bruno-366, I agree that putting manually links `#### <a name="foo"></a>` everywhere is kind of ugly.
On the one hand, I liked the underlying idea of making the *Optional* keyword more visible.
On the other hand, *all* docker-coq-action fields are actually optional :wink:, and the current version of the PR is broken (the internal links do not work anymore).
So I'd just suggest you rephrase all H4 headings by putting the ***Optional*** keyword like this:
```
#### `ocaml_version`

***Optional***

The version of OCaml.

[…]
```
Can you update the PR accordingly? and I'll merge it :relieved: